src/data-table/data-table.js

```diff
@@ -616,10 +620,12 @@ export function Unstable_DataTable(props: DataTablePropsT) {
   );
   const normalizedWidths = React.useMemo(() => {
     const sum = ns => ns.reduce((s, n) => s + n, 0);
-    const resizedWidths = measuredWidths.map((w, i) => w + resizeDeltas[i]);
+    const resizedWidths = measuredWidths.map(
+      (w, i) => Math.floor(w) + Math.floor(resizeDeltas[i]),
+    );
     if (gridRef.current) {
       // $FlowFixMe
-      const domWidth = gridRef.current.props.width;
+      const domWidth = gridRef.current.props.width - 2;
```

Comment:
may worth adding a comment on why we need this?

Reply:
```suggestion
      // minus 2 to account for the border stroke width
      const domWidth = gridRef.current.props.width - 2;
```